fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -32,14 +32,17 @@ void InitializePaths()
 {
     // For path comparison's sake - and the fact that std::filesystem::path doesn't handle (root-)local device paths all
     // that well - ensure that these paths are drive-absolute
-    auto packageRootPath = ::PSFQueryPackageRootPath();
-    if (auto pathType = psf::path_type(packageRootPath);
+    auto packageRootPath = std::wstring(::PSFQueryPackageRootPath());
+    if (auto pathType = psf::path_type(packageRootPath.c_str());

```

Comment:
Should the if statement be on the next line and have the assignment here always?